app/controllers/api/deploys_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::DeploysController < Api::BaseController
+  include CurrentProject
+
+  skip_before_action :require_project, only: [:active_count]
+
+  def index
+    scope = current_project.try(:deploys) || Deploy
+    @deploys = paginate(params[:ids].present? ? [scope.find(params[:ids])] : scope)
```

Comment:
this would produce an array in an array ... add a test ;)
```Ruby
@deploys = current_project.try(:deploys) || Deploy
if ids = paginate(params[:ids].presence
  @deploys = @deploys.find(ids.split(','))
end
```